core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1301,6 +1301,39 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_BACKUP_STATE_LOCK_EXCLUSIVE_DURATION =
+      new Builder(Name.MASTER_BACKUP_STATE_LOCK_EXCLUSIVE_DURATION)
+          .setDefaultValue("0ms")
+          .setDescription("Alluxio master will allow only exclusive locking of "
+              + "the state-lock for this duration. This duration starts after masters "
```

Comment:
I think we should be more explicit here. This will reject all RPCs which need to take the state-lock, which is most client RPCs.

Should we also mention that this can be useful to guarantee a window on restart to initiate a backup?